### PR TITLE
rebuild website

### PR DESCRIPTION
**Pull request by ghost**

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -1,46 +1,44 @@
...
+			<div class="justify-content-center my-auto">
+				<a href="https://github.com/l3nnartt" class="me-3 text-white"><i class="bi bi-github"></i></a>
+				<a href="https://twitter.com/l3nnartt" class="me-3 text-primary"><i class="bi bi-twitter"></i></a>
+				<a href="https://open.spotify.com/user/g4fa1a33h626g3ztl23p0gys6"
+					class="me-3 text-success w-100 me-4"><i class="bi bi-spotify"></i></a>
 			</div>
 		</div>
-	</body>
+		<div class="text-white-50 one mx-4 mt-4 pb-3">
+			<p class="fw-bold">ABOUT ME</p>
```
Comment (ghost) [inline index.html]: Instead of hard code the whole sentence, use the bootstrap class "text-uppercase". Also useful for SEO.

Comment (ghost) [reply index.html]: just forgot to implement that

**Comment by l3nnartt** [comment]
Looks very nice and modern but the background is kind of broken
image

**Comment by ghost** [comment]
fixed the background and did something with api.

**Comment by ghost** [comment]
switched to an even better solution for banner support.

**Comment by ghost** [comment]
lennartloeschede

**Comment by l3nnartt** [comment]
Thank you!

**Pull request merged**